<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="9" column="0">
     <widget class="QPushButton" name="importButton">
      <property name="text">
       <string>Import</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="addObstacleButton">
      <property name="text">
       <string>Add Obstacle</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="goButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Go</string>
      </property>
     </widget>
    </item>
    <item row="19" column="0" rowspan="17">
     <widget class="QTextEdit" name="textEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>578</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="1" rowspan="34" colspan="4">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="10" column="0">
     <widget class="QPushButton" name="exportButton">
      <property name="text">
       <string>Export</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="rotateCCButton">
      <property name="text">
       <string>Rotate Left</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="animStopButton">
      <property name="text">
       <string>Stop Animation</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="rotateCButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Rotate Right</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="stopButton">
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item row="18" column="0">
     <widget class="QPushButton" name="loadImportButton">
      <property name="text">
       <string>Load Scene</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="deleteLastObstacleButton">
      <property name="text">
       <string>Delete Last Obstacle</string>
      </property>
     </widget>
    </item>
    <item row="17" column="0">
     <widget class="QPushButton" name="saveImportButton">
      <property name="text">
       <string>Save import</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="addRobotButton">
      <property name="text">
       <string>Add Robot</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="animResButton">
      <property name="text">
       <string>Resume Animation</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="deleteLastRobotButton">
      <property name="text">
       <string>Delete selected Robot</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1061</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
